src/api/db/migrate/20200625142212_create_data_migrations.rb

```diff
@@ -0,0 +1,11 @@
+# rubocop:disable Rails/CreateTableWithTimestamps
+class CreateDataMigrations < ActiveRecord::Migration[6.0]
+  def up
+    return if ActiveRecord::Base.connection.table_exists? 'data_migrations'
```

Comment:
This can be replaced by `if_not_exists: true` in the `create_table` statement below.

https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-create_table

Reply:
nice